ui/components/MeshModelRegistry/MeshModelDetails.js

```diff
@@ -27,7 +27,12 @@ import {
 import _ from 'lodash';
 import { JustifyAndAlignCenter } from './MeshModel.style';
 import { withSuppressedErrorBoundary } from '../General/ErrorBoundary';
-
+import ReactJson from 'react-json-view';
+import { reactJsonTheme } from './helper';
+import Accordion from '@material-ui/core/Accordion';
+import AccordionDetails from '@material-ui/core/AccordionDetails';
+import AccordionSummary from '@material-ui/core/AccordionSummary';
+import ExpandMoreIcon from '@mui/icons-material/ExpandMore';
```

Comment:
why are we not using sistent .

Reply:
please use sistent . we constantly regress back and end up churning too much.  . 
All new and refactored components should strictly use sistent .

Reply:
I will keep that in my mind.